docs/install/deploy-gke-multi.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# Change the following variables to match your setup
+cluster1=k8ssandra-testing-dc1
+zone1=us-central1-c
+cluster2=k8ssandra-testing-dc2
+zone2=us-west1-a
+gcp_project=community-ecosystem
+
+# Add kubeconfig entries for both clusters
+gcloud container clusters get-credentials $cluster1 --zone $zone1 --project $gcp_project
+gcloud container clusters get-credentials $cluster2 --zone $zone2 --project $gcp_project
+
+# Install cert manager on both clusters
+kubectx gke_${gcp_project}_${zone1}_${cluster1}
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.3.1/cert-manager.yaml
+kubectx gke_${gcp_project}_${zone2}_${cluster2}
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.3.1/cert-manager.yaml
+
+# Install the k8ssandra control plane
+kubectx gke_${gcp_project}_${zone1}_${cluster1}
+kustomize build aux-config/control_plane | kubectl apply -f -
+
+# Install the k8ssandra data plane
+kubectx gke_${gcp_project}_${zone2}_${cluster2}
+kustomize build aux-config/data_plane | kubectl apply -f -
+
+# Create the client config
+export KUBECONFIG=./build/kubeconfigs/${cluster1}.yaml
+gcloud container clusters get-credentials ${cluster1} --zone ${zone1} --project ${gcp_project}
+export KUBECONFIG=./build/kubeconfigs/${cluster2}.yaml
+gcloud container clusters get-credentials ${cluster2} --zone ${zone2} --project ${gcp_project}
+unset KUBECONFIG
+
+scripts/create-clientconfig.sh --src-kubeconfig build/kubeconfigs/${cluster2}.yaml --dest-kubeconfig build/kubeconfigs/${cluster1}.yaml --in-cluster-kubeconfig build/kubeconfigs/${cluster2}.yaml --output-dir clientconfig
+
+# Restart the k8ssandra-operator pod on the control-plane
+kubectx gke_${gcp_project}_${zone1}_${cluster1}
+kubectl delete pod -l control-plane=k8ssandra-operator
+
+# Create a K8ssandraCluster
+cat <<EOF | kubectl apply -f -
+apiVersion: k8ssandra.io/v1alpha1
+kind: K8ssandraCluster
+metadata:
+  name: demo
+spec:
+  cassandra:
+    cluster: demo
+    serverVersion: "4.0.0"
+    serverImage: k8ssandra/cass-management-api:4.0.0-v0.1.29
```

Comment:
Do we want the `serverVersion` and `serverImage` to be setup variables for flexibility as well?